src/SDL_internal.h

```diff
@@ -91,6 +91,13 @@
 #elif defined(HAVE_MALLOC_H)
 #include <malloc.h>
 #endif
+#ifdef HAVE_STDBOOL_H
+#include <stdbool.h>
+#else
+typedef int bool;
+#define true    1
+#define false   0
+#endif
```

Comment:
I guess this block, including stdbool.h needs protecting against c++ compilers, i.e. `#ifndef __cplusplus` .. `#endif` ? IIRC, stdbool.h is not part of c++, at least gcc and compatibles allows it, but no guarantees. (I may be wrong though.)

Reply:
Good point. Added, thanks!

Reply:
`bool` is also a keyword in C23!